index.html

```diff
@@ -214,13 +214,14 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
             <input class="userinput" type="text" name="name" id="userName" maxlength="30" placeholder="Full name" required>
             <input class="userinput" type="email" name="email" id="userEmail" placeholder="Email address" required>
             <textarea class="userinput" name="message" id="userMessage" cols="30" rows="10" maxlength="500" placeholder="Enter your message" required></textarea>
+            <button id="sendContactMeForm" class="button" type="submit" form="contactmeForm">Get In touch</button>
```

Comment:
You should change the border color for all of your inputs in this form (light gray color), remember the green border is just when you select (focus) one of the inputs or text areas in the form. To access the properties in CSS when you are focusing  an input, try to use the next one:
```
input:focus {
  border-color: green;
}
```

- [ ] Fix border color in your form inputs.

Figma design | Yours

![image](https://user-images.githubusercontent.com/60164239/130098038-ef084046-9dda-4b5c-8683-929b34d04ecd.png)